plugins/snippets/nvim-snippets.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "nvim-snippets";
+  luaName = "snippets";
+  defaultPackage = pkgs.vimPlugins.nvim-snippets;
+
+  maintainers = [ lib.maintainers.psfloyd ];
+
+  settingsOptions = with helpers.nixvimTypes; {
+    create_autocmd = helpers.defaultNullOpts.mkBool false "Optionally load all snippets when opening a file. Only needed if not using nvim-cmp.";
```

```suggestion
    create_autocmd = helpers.defaultNullOpts.mkBool false ''
      Optionally load all snippets when opening a file.
      Only needed if not using nvim-cmp.
    '';

```

Please, adopt the same coding style as the rest of the code base. This ensure consistency across the project.